src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -396,7 +409,14 @@ private Pod getPodTemplate(KubernetesSlave slave, Label label) {
         volumes.add(new VolumeBuilder().withName(WORKSPACE_VOLUME_NAME).withNewEmptyDir("").build());
         Map<String, Container> containers = new HashMap<>();
 
+        if (template.isCustomJnlpContainerEnabled() && template.getJnlpContainer() != null) {
```

Comment:
> Even, if we add an option for that. I think that it wouldn't work, because you can't have a slave pod without a jnlp container.

sure, i just was thinking of the case where i don't want to call my container "jnlp" but it does indeed provide the jnlp client.

but it's not a big deal, if naming the container jnlp is the accepted mechanism i think we can live with it.